packages/design-tokens/.gitignore

```diff
@@ -0,0 +1 @@
+*.css
```

Comment:
This is so CSS that get into the NPM package don't make it into GIT?

Reply:
Correct! We'll be able to place the generated CSS file(s) in the `/dist` folder once we switch the package back to the [`exports`](https://nodejs.org/api/packages.html#exports) field.